### PR TITLE
Disable ports and move ports pkgs

### PR DESCRIPTION
**Pull request by Ornias1993**
TL:DR
- Comments out adding ports mounts to jail
- Removes unneeded ports install commands
- Adds two missing dependencies (according to nextcloud documentation)
- Moves the current ports downloads to pkgs

Long story:
One package are adviced by the Nextcloud documentation to be installed, it's little work to just add it: "php73-bcmath"

One package should actually be required to use APCu caching but isn't currently included. I just added it, because it's no work at all:
"php73-pecl-APCu"

Two Packages are installed using ports. But are currently available to be installed using pkgs and work fine. Installing them using pkgs instead of ports speeds up install significantly.

The jail side of ports commands are commented out. They are not removed to make it as easy as possible to use them again if needed in the future.

### CODE REVIEW

**Comment by danb35** [comment]
At the time, there was a problem installing those packages using the pkg, though it's been a while ago so I don't remember the exact problem.  They install and work properly using the pkg now?

**Comment by Ornias1993** [comment]
@danb35 
Well, I applied the same fix here: https://github.com/jailmanager/jailman/pull/130

And it installs fine, I can login fine and I can browse the basic apps+settings fine...
So I think I can say: I tested it and it works fine...
